### ISSUE TITLE
Cannot access hotInstance from cells function

### ISSUE THREAD

**Issue by aclark09** [handsontable/vue-handsontable-official#119]
### Description
When I try to add a cells function for conditional formatting, the handsontable instance is undefined. I was expecting to be able to access the instance from within this function so that I could check the value of a cell and use a different renderer based on that value. I was following this tutorial and attempting to recreate it with the vue wrapper: https://handsontable.com/docs/7.0.2/demo-conditional-formatting.html.

### Steps to reproduce
1. I've included a codepen that reproduces the issue

### Demo
https://codepen.io/anon/pen/mYbEaE

### Your environment
* Vue wrapper version: 4.0.0
* Handsontable version: 7.0.2
* Browser Name and version: Google Chrome 74.0.3729.131
* Operating System: Windows 10

**Comment by wojciechczerniak** [comment]
True. Handsontable instance will be set after the table is initialized, when `new Handsontable()` is finished. We can probably improve this if we switch to `new Handsontable.Core()` and `this.hotInstance.init()` inside the wrapper.

Like we did for `@handsontable/angular` recently: https://github.com/handsontable/angular-handsontable/pull/152/files#diff-e2d583e49d3e68f25b1b0818ad3ce755R334

For now you can move the function into settings object. Where instance is available through `this` context:

```
cells: function(row, column, prop) {
  const hotInstance = this.instance;
}
```

**Comment by AMBudnik** [comment]
Thank you for sharing your feedback @aclark09 
we're glad to announce that it got fixed in v `4.1.0` released yesterday.
